code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -41,8 +43,10 @@
 #undef HUMAN_EATING_NBP_MOUTH
 #undef HUMAN_EATING_BLOCKED_MOUTH
 
-/mob/living/carbon/human/proc/update_equipment_vision()
+/mob/living/carbon/human/proc/update_equipment_protection()
```

Comment:
make proc name more descriptive, because it updates only audiovisual protection, not general protection